src/docs/patterns/visualizations.mdx

```diff
@@ -1,62 +1,93 @@
+There are many opportunities to visualize data in different applications. The most common visualizations used in PX Blue applications come in the form of charts / graphs and maps.
+
+<Divider />
+
+# Charts / Graphs
+
+<ImageGrid
+    images={[LineChart, PieChart]}
+    caption={'Line charts (#1) and pie / ring charts (#2) are the most frequently used data visualizations in PX Blue.'}
+/>
+
+There are [many variations](https://material.io/design/communication/data-visualization.html#types) of data visualizations. They can be a big line chart with 8 series, occupying the entire screen; or they can be as tiny as a simple pie chart embedded in [lists](/patterns/lists).
 
 <MaterialDesignDescription
     title={'Data Visualization'}
     url={'https://material.io/design/communication/data-visualization.html'}
 />
 
-## Charts & Graphs (Highcharts & ChartJS)
-
-Currently, PX Blue supports building charts and graphs using [Highcharts](http://www.highcharts.com) and [ChartJS](https://www.chartjs.org/), which are comprehensive libraries compatible with both Angular and React.
-
-> **NOTE:** the following examples merely serve to show how to bring the charting libraries into your application and adjust some of the common chart parameters. You will want to adjust these values to meet the design requirements of your application.
-
-### Highcharts
+> The following examples merely serve to show how to bring the charting libraries into your application and adjust some of the common chart parameters. You will want to adjust these values to meet the design requirements of your application.
 
 <DemoCard
     repository={'highcharts'}
     angular={`https://stackblitz.com/github/pxblue/highcharts/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/highcharts/tree/master/react-demo`}
 />
-
-PX Blue utilities for Highcharts graphs (line, column, pie, donut) are available as an NPM package ([@pxblue/highcharts](https://www.npmjs.com/package/@pxblue/highcharts)).
-This package includes helper functions to generate chart configurations using recommended PX Blue styles.
-These utilities also enable you to quickly create placeholder charts using simulated data. For more information about using Highcharts, check out their documentation.
-
-> Highcharts has different [licensing](https://shop.highsoft.com/developer-license) requirements depending on the nature of your application - you will need to contact Highsoft directly to ensure that your product is properly licensed. You can also see the [Highcharts Pricing](https://shop.highsoft.com/)) page for pricing information.
-
-### ChartJS
-
 <DemoCard
     repository={'chartjs'}
     angular={`https://stackblitz.com/github/pxblue/chartjs/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/chartjs/tree/master/react-demo`}
 />
 
-ChartJS is an open source alternative to HighCharts offering a scaled-down feature set that supports basic charting needs.
+## Common Pitfalls
```

Comment:
There's only one sentence in this sections (and I don't know that I would necessarily call it a pitfall). I would just move that sentence up and have it under the umbrella of Charts/Graphs for now.